.github/benchmark_configs/pr_benchmark.json

```diff
@@ -27,6 +29,25 @@
     ],
     "cluster_mode": true,
     "tls_mode": false,
-    "warmup": 10
+    "warmup": 180,
+    "server_cpu_range": "0",
+    "client_cpu_range": "55,95"
+  },
+  {
+    "requests": [100000000000],
+    "keyspacelen": [3000000],
+    "data_sizes": [512],
+    "pipelines": [10],
+    "clients": [650],
+    "commands": [
+      "SET",
+      "GET"
+    ],
+    "cluster_mode": false,
+    "tls_mode": false,
+    "warmup": 300,
+    "io-threads": 8,
+    "server_cpu_range": "0-7",
+    "client_cpu_range": "55,95"
```

Comment:
What's the reasoning behind this scenario? Could we add comments to help explain them to others looking at this.